packages/notifi-react-card/lib/constants/countryData.tsx

```diff
@@ -0,0 +1,137 @@
+export type CountryMetadata = {
+  name: string;
+  dialCode: string;
+  flag: string;
+};
+
+type CountryMap = {
+  [countryCode: string]: CountryMetadata;
+};
+
+export const countryMap: CountryMap = {
```

Comment:
I think `[countryCode: string]` is a weak constraint -- it allows me to invoke invalid keys, like `countryMap['hello world']`.

Consider exporting `countryMap = { ... } as const`, `type CountryMap = typeof countryMap`, and `type CountryMetadata = CountryMap[keyof CountryMap]` instead.